samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -323,6 +324,13 @@ static const char link_redmob_usage_str[] =
 	"      --default,      Enable default reduced mobility mode\n"
 	"      --nordic,       Enable Nordic proprietary reduced mobility mode\n";
 
+static const char link_propripsm_usage_str[] =
+	"Usage: link propripsm --read | --disable | --enable\n"
```

Comment:
I did think about this as well. I'm leaning towards not grouping all features under the same command, because that would be a mixed bag of unrelated features. The only benefit would be that it would be more aligned with the AT commands, I think.

Reply:
But do we want ~5 different subcommands for different features?

Reply:
Why not? And currently there are only two features in FEACONF.